### PR TITLE
TECH-6367-windows-provisioning-pip-version

### PR DESCRIPTION
**Pull request by raj-wh**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -184,11 +184,17 @@
...
+        exec { 'pip-install':
+          command => "${easy_install} pip==20",
+          creates => $pip_script,
+          require => $setuptools_require,
+        } } else { exec { 'pip-install':
+          command => "${easy_install} pip",
+          creates => $pip_script,
+          require => $setuptools_require,
+        } }
+
```
Comment (jdavisp3) [inline manifests/init.pp:197]: I would suggest we do this a bit differently.

1. Default a class variable called `easy_install_pip` that defaults to `pip`.
2. Make the `exec` block above use `command => "${easy_install} ${easy_install_pip}"`.

That way there's no change in behavior for this public package but we we can can the pip version via Hiera.

**Review by jdavisp3 — CHANGES_REQUESTED** [review]

**Review by raj-wh — COMMENTED** [review]
Made pip a class variable

File: manifests/init.pp [hunk 2]
```diff
@@ -57,6 +57,7 @@
   $pip_ensure         = 'installed',
   $pip_package        = $python::params::pip,
   $setuptools_ensure  = 'installed',
+  ${easy_install_pip} = 'pip',
```
Comment (jdavisp3) [inline manifests/init.pp:60]:
```suggestion
  $easy_install_pip = 'pip',
```

When you declare a pip variable or use it outside of a string you don't use the `{}`. Braces are for string interpolation.

**Review by jdavisp3 — CHANGES_REQUESTED** [review]

**Review by jdavisp3 — APPROVED** [review]
LGTM!

**Comment by raj-wh** [comment]
@jdavisp3 I don't seem to have merge permission on this repo.

**Pull request merged**